src/main/scala/is/hail/annotations/aggregators/RegionValueAggregator.scala

```diff
@@ -11,5 +11,7 @@ trait RegionValueAggregator extends Serializable {
 
   def copy(): RegionValueAggregator
 
+  def deepCopy(): RegionValueAggregator
```

Comment:
Can you clarify why/whether we need the non-deep copy() ?  Off the top of my head, it seems that at least any mutable data reachable from an Aggregator
would always need to be copied - and that the sharing of immutable data (e.g.
strings ?) would an optimization that doesn't need to be apparent in the interface.

In other words, don't we *always* want the semantics to be "as-if deep copy" ?

Or are we using the "copy()" method to mean something else altogether,
viz "make a new empty Aggregator of the same type as this one" ?  But if that's what we mean, I think it shouldn't be called "copy()" but
emptyCopy(), or something else.